generators/entity/index.js

```diff
@@ -273,7 +273,14 @@ module.exports = EntityGenerator.extend({
         if (this.relationships.length > 0) {
             this.log(chalk.white('Relationships'));
             this.relationships.forEach(function (relationship) {
-                this.log(chalk.red(relationship.relationshipName) + ' ' + chalk.white('(' + _.upperFirst(relationship.otherEntityName) + ')') + ' ' + chalk.cyan(relationship.relationshipType));
+                var validationDetails = '';
+                var relationshipValidate = relationship.relationshipValidateRules && relationship.relationshipValidateRules.length >= 1;
+                if (relationshipValidate === true) {
```

Comment:
can you simplify this? I guess i gave this comment in your old PR as well. can you please update same PR in future instead of opening new ones as its really inconvenient to review.

```
if (relationship.relationshipValidateRules && relationship.relationshipValidateRules.indexOf('required') !== -1)
```
only this condition would be sufficient to replace three lines of your code